base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -3985,7 +4015,7 @@ InitPropertiesDlg(HWND hDlg, PEVENTLOG EventLog)
 
     if (EventLog->Permanent)
     {
-        SendDlgItemMessageW(hDlg, IDC_UPDOWN_MAXLOGSIZE, UDM_SETRANGE32, (WPARAM)1, (LPARAM)0x3FFFC0);
+        SendDlgItemMessageW(hDlg, IDC_UPDOWN_MAXLOGSIZE, UDM_SETRANGE32, (WPARAM)64, (LPARAM)0x3FFFC0);
```

Comment:
When I coded that thing I wanted a finer granularity than 64kb jumps. So please revert. There is no point in adding artificial limitations.